acceptance/go.mod

```diff
@@ -30,10 +30,7 @@ require (
 )
 
 // replace these so we always use the latest version of the control-plane types
-replace (
-	github.com/hashicorp/consul-k8s/control-plane => ../control-plane
-	github.com/hashicorp/consul-k8s/version => ../version
-)
+replace github.com/hashicorp/consul-k8s/version => ../version
```

Comment:
If we don't do this, then acceptance cannot differ for k8s.io versions. I did bump consul-k8s control plane though

Reply:
Can we not keep the replace for `control-plane` so we're always testing current for PRs that change it? Or was this made necessary for now by k8s.io / some other thing I'm maybe overlooking

Reply:
It's the k8s.io version unfortunately! It will keep trying to pull in k8s.io 0.29.8 if we follow exactly. So we will need a fairly fast follow to actually get acceptance on k8s.io 0.29.8 if we want to bring back this replace. 

I think since PSPs don't work after K8s 1.25 so it's mostly a matter of removing that config when bumping k8s.io but I just didn't want to add on to an already hairy yak shave which is why I chose to not do it right before the release.

Reply:
Got it. I think that fast follow is important here so we can keep regression testing strong before 1.20 and Oct. patches. Thanks for being careful and explaining!